docs/src/main/asciidoc/kafka-reactive-getting-started.adoc

```diff
@@ -488,32 +491,36 @@ services:
       KAFKA_LISTENERS: PLAINTEXT://0.0.0.0:9092
       KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
     networks:
-      - kafkaquickstart-network
+      - kafka-quickstart-network
 
   producer:
     image: quarkus-quickstarts/kafka-quickstart-producer:1.0-${QUARKUS_MODE:-jvm}
     build:
       context: producer
       dockerfile: src/main/docker/Dockerfile.${QUARKUS_MODE:-jvm}
+    depends_on:
+      - kafka
     environment:
       KAFKA_BOOTSTRAP_SERVERS: kafka:9092
     ports:
       - "8080:8080"
     networks:
-      - kafkaquickstart-network
+      - kafka-quickstart-network
 
   processor:
     image: quarkus-quickstarts/kafka-quickstart-processor:1.0-${QUARKUS_MODE:-jvm}
     build:
       context: processor
       dockerfile: src/main/docker/Dockerfile.${QUARKUS_MODE:-jvm}
+    depends_on:
+      - kafka
     environment:
       KAFKA_BOOTSTRAP_SERVERS: kafka:9092
     networks:
-      - kafkaquickstart-network
+      - kafka-quickstart-network
 
 networks:
-  kafkaquickstart-network:
+  kafka-quickstart-network:
```

Comment:
That was also proposed by @cescoffier

Reply:
No big preference. I slightly prefer the kebab variant.

Reply:
I did this to bring the documentation and the quickstart in sync. I can sync them in the other direction for sure, but this looks better to me :-)